test/general/sycl_iterator/sycl_iterator_for.pass.cpp

```diff
@@ -243,10 +256,26 @@ DEFINE_TEST(test_uninitialized_value_construct_n)
     }
 };
 
+template <typename Iterator>
+using __it_test_type = decltype(dpl::begin(::std::declval<sycl::buffer<typename std::iterator_traits<Iterator>::value_type>>()));
+
 DEFINE_TEST(test_destroy)
 {
     DEFINE_TEST_CONSTRUCTOR(test_destroy)
 
+    template <typename Policy, typename Iterator1, typename Size>
+    void
+    operator()(Policy&& exec, __it_test_type<Iterator1> first1, __it_test_type<Iterator1> last1, Size n)
+    {
+	if constexpr (std::is_trivially_copyable_v<TestValueType>)
```

Comment:
Incorrect formatting

Reply:
The same like in https://github.com/oneapi-src/oneDPL/pull/1307/files#r1415847734

Reply:
Actually,
```
@github-actions
oneDPL CI / clang-format (pull_request) Successful in 11s
```